Tasks/JavaToolInstallerV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -21,9 +21,11 @@
   "loc.input.label.azureCommonVirtualFile": "Common virtual path",
   "loc.input.help.azureCommonVirtualFile": "Path to the JDK inside the Azure storage container.",
   "loc.input.label.jdkDestinationDirectory": "Destination directory",
-  "loc.input.help.jdkDestinationDirectory": "On Linux and Windows, this is used as the destination directory for JDK installation. On macOS, this directory is used as a temporary folder for extracting .dmg's since macOS doesn't support installing of JDK to specific directory.",
+  "loc.input.help.jdkDestinationDirectory": "On Linux and Windows, this is used as the destination directory for JDK installation. On macOS, this directory is used as a temporary folder for extracting of .dmg's since macOS doesn't support installing of JDK to specific directory.",
```

Comment:
Is this intentional? :)

Reply:
It is autogenerated file, so seems like when these changes introduced initially in task.json this file wasn't regenerated.